tests/go_test.go

```diff
@@ -98,6 +99,54 @@ func TestTextParsing(t *testing.T) {
 	}
 }
 
+func CheckPizza(fail func(string, ...interface{})) {
+	var empty, nonempty []byte
+
+	// create food with an empty pizza
+	{
+		builder := flatbuffers.NewBuilder(0)
+
+		order.FoodStart(builder)
+		m := order.FoodEnd(builder)
+		builder.Finish(m)
+
+		empty = make([]byte, len(builder.FinishedBytes()))
+		copy(empty, builder.FinishedBytes())
+	}
+
+	// create food with a non-empty pizza field
+	{
+		builder := flatbuffers.NewBuilder(0)
+		mygame.InParentNamespaceStart(builder)
+		pn := mygame.InParentNamespaceEnd(builder)
+
+		order.FoodStart(builder)
+		order.FoodAddPizzaTest(builder, pn)
+		m := order.FoodEnd(builder)
+
+		builder.Finish(m)
+
+		nonempty = make([]byte, len(builder.FinishedBytes()))
+		copy(nonempty, builder.FinishedBytes())
+	}
+
+	// read food with empty pizza field
+	{
+		m := order.GetRootAsFood(empty, 0)
+		if m.PizzaTest(nil) != nil {
+			fail("expected nil ParentNamespaceTest for empty field")
+		}
+	}
+
+	// read food with non-empty pizza field
+	{
+		m := order.GetRootAsFood(nonempty, 0)
+		if m.PizzaTest(nil) == nil {
+			fail("expected non-nil ParentNamespaceTest for non-empty field")
+		}
+	}
+}
```

Comment:
Instead of testing the change like this I think we should just do a happy path to make sure everything is imported correctly and builds.

1. Create a builder
2. Add pizza to builder with some values
3. Add order to builder with pizza stored in it
4. Init the order
5. Verify that the pizza data stored in the order is expected